charts/console/templates/platform/deployment.yaml

```diff
@@ -69,7 +69,6 @@ spec:
       securityContext: {{- include "common.tplvalues.render" (dict "value" .Values.platform.containerSecurityContext "context" $) | nindent 10 }}
       {{- else }}
       securityContext:
-        readOnlyRootFilesystem: true
```

Comment:
ok to leave on container ?

Reply:
It is indeed, I believe it's not only on podSecurityContext for some distribution but on all for container